server/middleware/validate.js

```diff
@@ -0,0 +1,14 @@
+const isValidObjectId = require("../utils/mongoObjectIdValidator");
+
+const validateMongoObjectId = (req, res, next) => {
+
+  const recordId = req.params.id;
+
+  if (!isValidObjectId(recordId)) {
+    return res.status(400).send(`Invalid Record Id: ${recordId}`);
+  }
```

Comment:
Sorry if I wasn't super clear but there's actually a library we have called https://express-validator.github.io/docs/ that I would like to use rather than making this from scratch.

Reply:
https://github.com/hatchways/team-bond/blob/main/server/validate.js 

Example file that we have in the server root

Reply:
`check().isMongoId` I think is the function we need to use.

Reply:
Thanks. I implemented it with the express-validator this time.